packages/fx-core/src/common/local/taskDefinition.ts

```diff
@@ -103,6 +103,45 @@ export class TaskDefinition {
     };
   }
 
+  static funcHostedBotWatch(workspaceFolder: string): ITaskDefinition {
+    return {
+      name: "bot watch",
+      // command is never used for the new local preview in CLI.
+      command: "npm run watch:teamsfx",
+      cwd: path.join(workspaceFolder, FolderName.Bot),
+      execOptions: {
+        needShell: true,
+        needCmd: isWindows(),
+      },
+      isBackground: true,
+    };
+  }
+
+  static funcHostedBotStart(workspaceFolder: string): ITaskDefinition {
+    return {
+      name: "bot start",
+      // command is never used for the new local preview in CLI.
+      command: `npm run dev:teamsfx`,
+      cwd: path.join(workspaceFolder, FolderName.Bot),
+      execOptions: {
+        needShell: true,
```

Comment:
`backendStart` has `needCmd`, what about `funcHostedBotStart`?

Reply:
see the last comment

Reply:
So the start task requires `needCmd`, but watch task does not.